src/index.html

```diff
@@ -4,9 +4,470 @@
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+              </div>
+
+              <p class="card__reviews">
+                Reviews: 5
+              </p>
+            </div>
+
+            <div class="card__price">
+              <p class="card__price-text">
+                Price:
+              </p>
+
+              <p class="card__price-amount">
+                &dollar;2,199
+              </p>
+            </div>
+
+            <a href="#" class="card__link">
+              Buy
+            </a>
+          </div>
+
+          <div class="card card--margin">
+            <img
+              class="card__product-img"
+              src="./images/imac.jpeg"
+              alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+            >
+
+            <h3 class="card__title">
+              APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)
+            </h3>
+
+            <p class="card__product-code">
+              Product code: 195434
+            </p>
+
+            <div class="card__stars">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+              </div>
+
+              <p class="card__reviews">
+                Reviews: 5
+              </p>
+            </div>
+
+            <div class="card__price">
+              <p class="card__price-text">
+                Price:
+              </p>
+
+              <p class="card__price-amount">
+                &dollar;2,199
+              </p>
+            </div>
+
+            <a href="#" class="card__link">
+              Buy
+            </a>
+          </div>
+
+          <div class="card card--margin">
+            <img
+              class="card__product-img"
+              src="./images/imac.jpeg"
+              alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+            >
+
+            <h3 class="card__title">
+              APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)
+            </h3>
+
+            <p class="card__product-code">
+              Product code: 195434
+            </p>
+
+            <div class="card__stars">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+              </div>
+
+              <p class="card__reviews">
+                Reviews: 5
+              </p>
+            </div>
+
+            <div class="card__price">
+              <p class="card__price-text">
+                Price:
+              </p>
+
```

Comment:
better change price tag to h2
```suggestion
            <div class="card__price">
              <h2 class="card__price-text">
                Price:
              </h2>

```